lib/Config.js

```diff
@@ -43,6 +43,14 @@ class Config {
             this.port = config.port;
         }
 
+        this.listenOn = [];
+        if (config.listenOn !== undefined) {
+            assert(Array.isArray(config.listenOn)
+                && config.listenOn.every(e => typeof e === 'string'),
```

Comment:
The best part this the container would be in "restarting" state with the worker crashing and restarting in a loop. So it would be apparent when they mess up.